canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -177,7 +177,7 @@ class ToolTip extends React.Component {
 				}
 			} else if (tooltipDirection === "left") {
 				// For long tooltips, tooltip.offsetWidth is updated after setting tooltip.style.left. Ensure tooltip doesn't overlap tooltipTrigger element.
-				while ((tooltip.offsetLeft + tooltip.offsetWidth + pointerLayout.width) > triggerLayout.left) {
+				while ((tooltip.offsetLeft + tooltip.offsetWidth + pointerLayout.width) > Math.round(triggerLayout.left)) {
```

Comment:
`tooltip.offsetLeft` is a whole number. So rounding off `triggerLayout.left` to get rid of decimal part which was causing the infinite loop.